cross-chain/arbitrum/contracts/ArbitrumTBTC.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: GPL-3.0-only
```

Comment:
I think there is little risk that we'll have to upgrade the token only for one chain. Anyway, if it happens we have three options:
- Add a storage gap to the `solidity/` implementation of `L2TBTC` and inherit from the respective cross-chain module.
- Copy-paste `L2TBTC` contract to an appropriate cross-chain module and change the code there. 
- Copy-paste `L2TBTC` to the respective cross-chain module, add storage gap, and inherit.

As long as we do not mess up with storage slots, we are fine.

It may mean, we will change the code in the copied version of `L2TBTC` without inheriting or add a gap and inherit. We can't predict if we'll be changing existing logic or adding new logic (your first point).

Reply:
Ok, I see the plan now. I was concerned about the `gap` but I agree we can't predict how and if that'll change. We can add it later just like you mentioned. Here's a change [8887512](https://github.com/keep-network/tbtc-v2/pull/554/commits/8887512a629448bbd6f72b4fd9eb42da3e493bf1)